Symfony/CS/Tokenizer/TokensAnalyzer.php

```diff
@@ -211,7 +211,79 @@ public function isArrayMultiLine($index)
     }
 
     /**
-     * Check if there is a lambda function under given index.
+     * Returns the attributes of the method under the given index.
+     *
+     * @param int $index Token index of the method (T_FUNCTION)
+     *
+     * @return array
+     */
+    public function getMethodAttributes($index)
+    {
+        $tokens = $this->tokens;
+        $token = $tokens[$index];
+
+        if (!$token->isGivenKind(T_FUNCTION)) {
```

Comment:
I hope this https://github.com/SpacePossum/PHP-CS-Fixer/commit/e482222f38b0c70cacae5a67f7e66607b2c7bceb#diff-fd0d877fbc0714da627810c6cffa4217R218  is OK?